components/infobox/extensions/commons/metadata_generator.lua

```diff
@@ -23,7 +23,7 @@ local TIME_FUTURE = 1
 local TIME_ONGOING = 0
 local TIME_PAST = -1
 
-local TYPES_TO_DISPLAY = {'qualifier', 'showmatch', 'show match'}
+local TYPES_TO_DISPLAY = {'qualifier', 'showmatch'}
```

Comment:
Are all wikis now without the space (storagewise at least)?

Reply:
`Tier.toIdentifier` removes the space (and the table is checked against the return of that)